src/extension/features/toolkit-reports/pages/balance-over-time/BalanceOverTime.jsx

```diff
@@ -4,6 +4,8 @@ import { FiltersPropType } from 'toolkit-reports/common/components/report-contex
 import { RunningBalanceGraph } from './RunningBalanceGraph';
 import { LabeledCheckbox } from 'toolkit-reports/common/components/labeled-checkbox';
 import { WarningMessage } from './WarningMessage';
+import { useLocalStorage } from '../../../../hooks/useLocalStorage';
+import { getEntityManager } from '../../../../utils/ynab';
```

Comment:
you can just use the absolute `toolkit` and `toolkit-reports` imports here so we don't have all the `dots`